sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,907 @@
+# Preamble
+
+SIP Number: 022
+
+Title: A Trustless 2-way Bitcoin Peg for Stacks
+
+Authors:
+* Aaron Blankstein <aaron@hiro.so>
+* Mike Cohen <mjoecohen@gmail.com>
+* Brice Dobry <brice@hiro.so>
+* Marvin Janssen
+* Friedger Müffke <mail@friedger.de>
+* Jesus Najera 
+* Jude Nelson <jude@stacks.org>
+* Don Park <donp@trustmachines.co>
+* Alie Slade
+* Andre Serrano <andre@resident.stacks.org>
+* @Soju-Drinker 
+* Igor Sylvester
+* Joey Yandle <joey@trustmachines.co> 
+
+Consideration: Technical, Governance
+
+Type: Consensus
+
+Status: Draft
+
+Created: 28 September 2023
+
+License: CC0-1.0
+
+Sign-off:
+
+Layer: Consensus (hard fork)
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+This SIP proposes the creation of a **trustless 2-way peg** in which BTC (the
+tokens of the current Stacks burnchain) can be manifested on Stacks as a
+SIP-010-compliant fungible token, called sBTC, and later redeemed for the same
+number of BTC.  This proposal redefines the act of Stacking to be the act of
+maintaining an _sBTC wallet_, which is used to hold BTC and process sBTC
+redemptions.  Stackers are collectively responsible for ensuring that all
+liquid sBTC are backed by an equal number of BTC, and that redemptions are
+processed in a timely manner.  As a reward, they continue to receive PoX
+payouts.
+
+# Introduction
+
+The lack of a stateful smart contract system on Bitcoin necessitates the
+construction of systems in which a representation of locked Bitcoin is traded
+within a separate smart-contract-powered blockchain.  These systems aim to
+provide a _2-way peg_ between the representation of the locked BTC ("wrapped
+BTC") and the BTC itself.  At a high level, these systems seek to provide two
+primitive operations:
+
+* "Deposit": a BTC holder rids themselves of their BTC, and in doing so,
```

Comment:
While I agree in the "deposit" & "withdraw" nomenclature, it'd be helpful to mention that the former can be thought of as "peg-in" & the latter as a "peg-out" in this definition section.

Especially since the term "peg" is used throughout.

Reply:
We're trying not to use the word "peg"

Reply:
We use the word "peg" in practice, so while it isn't **technically** the best word, its a practical word. I'm generally pro-practical.